.bumpversion.cfg

```diff
@@ -18,42 +18,44 @@ serialize =
 
 [bumpversion:file:airbyte-metrics/reporter/Dockerfile]
 
-[bumpversion:file:airbyte-server/Dockerfile]
+[bumpversion:file:airbyte-proxy/Dockerfile]
```

Comment:
for my understanding, the changeset in this file is
1) sorting (thanks!)
2) adding the airbyte proxy dockerfile

is that right?